src/sqlancer/postgres/ast/PostgresBinaryArithmeticOperation.java

```diff
@@ -58,15 +58,21 @@ public PostgresConstant apply(PostgresConstant left, PostgresConstant right) {
         private String textRepresentation;
 
         private static PostgresConstant applyBitOperation(PostgresConstant left, PostgresConstant right,
```

Comment:
This function name seems a bit confusing. Should it be `applyBinaryOperation`?

Reply:
You are right, or perhaps `applyBinaryArithmeticOperation` - either would be better.